### PR TITLE
Always add rhs property

### LINKED ISSUE #96: Extended Attributes `rhs` should always be there
It's somewhat annoying to have to check for missing properties, as is the case for `rhs` on extended attributes. 

We should make sure we don't conditionally add/remove members - and where they are null, they should just be `null`.

### PR DESCRIPTION
**Pull request by SaschaNaz**
Fixes #96.

### CODE REVIEW

File: lib/webidl2.js [hunk 1]
```diff
@@ -343,32 +343,28 @@
         name: name.value,
         "arguments": null,
         type: "extended-attribute",
+        rhs: null
```
Comment (marcoscaceres) [inline lib/webidl2.js]: I'm now wondering if we should just make it an empty list?

Comment (marcoscaceres) [reply lib/webidl2.js]: actually, ignore this.

**Comment by marcoscaceres** [comment]
Shipped at 8.1.0 :)

**Comment by SaschaNaz** [comment]
>Generally speaking, and from a functional perspective, it's quite frustrating to deal with when you sometimes get nothing, sometimes an object, other times an array :(

I agree, could you file an issue for that?

**Comment by marcoscaceres** [comment]
filed: 
https://github.com/w3c/webidl2.js/issues/113

**Pull request merged**